examples/preprocessing/plot_power_transformer.py

```diff
@@ -0,0 +1,85 @@
+"""
+==========================================================
+Using PowerTransformer to apply the Box-Cox transformation
+==========================================================
+
+This example demonstrates the use of the Box-Cox transform
+through ``PowerTransformer`` to map data from various
```

Comment:
use ```:class:`preprocessing.PowerTransformer` ```

Reply:
You can also modify the previous example accordingly.